frontend/src/app/sessions/user-sessions-wrapper/active-sessions/file-browser-dialog/file-browser-dialog.component.html

```diff
@@ -99,6 +99,15 @@ <h1 class="text-lg font-bold">Session File Browser</h1>
             <mat-icon>download_file</mat-icon>
           </button>
         </div>
+        <div class="ml-1 flex w-8">
```

Comment:
```suggestion
        <div class="flex w-8">
```

`ml-1` is only used to add some margin next to the text, but not required for the second button.